.github/CODEOWNERS

```diff
@@ -126,6 +126,7 @@
 /ns1/                                    @Zach-Johnson zjohnson@ns1.com @DataDog/ecosystems-review
 /nvml/                                   @cep21
 /octoprint/                              @gwaldo gwaldo@gmail.com
+/onepane/                                @sajosam sajo@onepane.ai @DataDog/ecosystems-review
```

Comment:
Please also add the other CODEOWNER entries in the other section in this file. (see all `/ns1` entries for example)